### ISSUE TITLE
folder-color-switcher Compatibility + generic icons

### ISSUE THREAD

**Issue by cain-dev** [vinceliuice/WhiteSur-icon-theme#306]
I've noticed you have made different versions for folders, but there is not file to make it compatible with Nemo's folder-color-switcher plugin. Could you please make it compatible?

Another thing that i find it very strange using it on my Cinnamon desktop is specific icons for generic applications, like, the "Finder" icon for my file manager instead of just a generic folder icon, it's strange 'cause i don't use "Finder" here, it's different if it's the "Mac version" for Firefox, 'cause i use Firefox on both systems.

In short, i don't think it's a request on NEW icons, but just for a new organization for the same icons.

Thank you for your work btw
